### ISSUE TITLE
What happen with nlast versions

### ISSUE THREAD

**Issue by Spiderbezno** [primefaces/primefaces#12]
The las jar available is 5.1.3, but official realese 5.1.8, are there any problem??

**Comment by brunolmfg** [comment]
Yes. My time is too short in recent months. I have a job, a wife and a daughter. ;-)

In my work the time available for this "project" was removed, so on the last releases I did everything I could to complete them.

I'm sorry, but I don't expect have time on the next weeks. :-(

Regards,
Bruno.

**Comment by ymajoros** [comment]
Hi Bruno,

Could you basically describe the release process? Is it just taking 'some'
revision and tagging it in git?

I suggest others fork your repo and send you pull requests, so you just
have to check and merge them back.

Best regards,

Yannick

2015-01-13 15:44 GMT+01:00 Bruno Gonçalves <notifications@github.com>:

**Comment by gulbinas** [comment]
:+1:

**Comment by brunolmfg** [comment]
Hi Yannick,

If there is something urgent to be done is document the overall process. I'll describe this process as soon as possible. Basically, the commits are sequentially paired with the issues tagged for the coming release or the changes identifying the related issue. If so they are merged, if not they are also merged but with "ours" strategy (git merge -s ours). Perhaps doing cherry-pick is more appropriate to distribute the work. I think.

However, the initial goal of the "project" was performing releases very close to official and not do with their own specific or different features and fixes. So, it's appropriate to create a different release cycle with its own features and fixes? Some features and fixes are not possible simply merging the changes from the Google Code repository. In these cases it's necessary to implement or adjust the changes ourselves because there are differences between the version in the trunk branch and the release branch.

Then left some questions:
* How many collaborators we have?
* Who would do the simple merging or cherry-picking?
* How distribute the "problematic" issues?

I think the simple merging (or cherry-picking) is a work for one person, maybe the same pushing the changes from the original trunk branch.

I want answers and your questions. ;-P


Best regards,
Bruno.

**Comment by huds8n** [comment]
Ola Bruno, vi que ultimamente anda sem tempo para o projeto envolvendo o primefaces. Não seria uma possibilidade colaborarmos com o projeto?? Obrigado !!!

**Comment by brunolmfg** [comment]
I'll synchronize the master branch with the trunk.
